### PR TITLE
Remove `.darts` directory after tests.

### PR DESCRIPTION
**Pull request by Kostiiii**
Fix minor issue with dangling `.darts` directory with checkpoints after running tests.

### Summary
TCNModel was creating `.darts` directory without removing it after tests.

### CODE REVIEW

File: darts/tests/test_TCN.py [hunk 1]
```diff
@@ -12,6 +13,10 @@ class TCNModelTestCase(unittest.TestCase):
     def setUpClass(cls):
         logging.disable(logging.CRITICAL)
 
+    @classmethod
```
Comment (TheMP) [inline darts/tests/test_TCN.py:16]: Why remove this directory only in TCN test case?

Comment (Kostiiii) [reply darts/tests/test_TCN.py:16]: I checked other test cases and the ones that use model that creates this directory already cleanup after themselves.

Comment (pennfranc) [reply darts/tests/test_TCN.py:16]: Yeah I forgot to add this, thanks @Kostiiii!

Comment (TheMP) [reply darts/tests/test_TCN.py:16]: Approved

**Pull request merged**